src/Tec/Plugin.php

```diff
@@ -139,61 +130,132 @@ protected function register_plugin_dependencies() {
 	}
 
 	/**
-	 * Get this plugin's options prefix.
-	 *
-	 * Settings_Helper will append a trailing underscore before each option.
-	 *
-	 * @return string
-     *
-	 * @see \Tribe\Extensions\Advanced_ICal_Export_V2\Settings::set_options_prefix()
+	 * Prepares the export feed based on the supplied parameters.
 	 *
-	 * TODO: Remove if not using settings
+	 * @param $repository_args
+	 * @param $context
+	 * @param $view
 	 */
-	private function get_options_prefix() {
-		return (string) str_replace( '-', '_', 'tribe-ext-advanced-ical-export-v2' );
-	}
+	public function custom_ical_export( $repository_args, $context, $view ) {
+	//public function custom_ical_export( $repository_args, $view ) {
 
-	/**
-	 * Get Settings instance.
-	 *
-	 * @return Settings
-	 *
-	 * TODO: Remove if not using settings
-	 */
-	private function get_settings() {
-		if ( empty( $this->settings ) ) {
-			$this->settings = new Settings( $this->get_options_prefix() );
+		// Sanitization
+		$vars = $this->getVars();
+
+		// Bail if not custom iCal export
+		if (
+			! tribe_context()->get( 'ical' )
+			&& $vars['custom'] != 1
```

Comment:
Avoid a potential notice. Use yoda conditional.

```suggestion
			&& ! empty( $vars['custom'] )
			&& 1 != $vars['custom']
```

Reply:
Right you are! :)